source/index.html.md.erb

```diff
@@ -61,12 +72,20 @@ specifically noted in this document; the key ID to use should be the first 8
 characters of the Base64 encoding of this key. The only supported algorithm is
 `hmac-sha256`, and the only supported Digest is `SHA-256`.
```

Comment:
Is it worth mentioning the dreaded `hs2019` here?

Reply:
I don't think it's worth worrying about - our implementation doesn't care what you pass in as an algorithm, and we pass hs2019 as the spec describes.